presto-verifier/src/main/java/com/facebook/presto/verifier/Validator.java

```diff
@@ -208,32 +214,74 @@ private boolean validate()
         // query has too many rows. Consider blacklisting.
```

Comment:
rename `executeQueryControl` to `executePreAndMainForControl`

Accordingly, rename tearDown to something like `executeTearDownForControl`